buildtest/cli/commands.py

```diff
@@ -1,8 +1,7 @@
-def list_buildtest_commands():
-    """This method implements command ``buildtest commands`` which shows a list
-    of buildtest commands"""
+def list_buildtest_commands(): """This method implements command ``buildtest commands`` which shows a list of buildtest commands"""
```

Comment:
put a new line after the method it looks like docstring is on same line as the method declaration. Something like this. Make sure its indented properly

```python
def list_buildtest_commands(): 
    """This method implements command ``buildtest commands`` which shows a list of buildtest commands"""
```